auto_nag/scripts/configs/tools.json

```diff
@@ -328,7 +314,16 @@
             "Blocklist Implementation",
             "Reader Mode",
             "Toolbars and Toolbar Customization",
-            "Framework"
+            "Framework",
+            "DevTools::General",
+            "Core::General",
+            "DOM: IndexedDB",
```

Comment:
Is it going to use @overholt's calendar?

Reply:
For Core::General: yes.

Reply:
ok, I hope he won't be mad with all the ni ;)